pom.xml

```diff
@@ -123,6 +137,14 @@
 			<optional>true</optional>
 		</dependency>
 		
+		<dependency>
+    		<groupId>org.springframework.data</groupId>
+    		<artifactId>spring-data-rest-hal-browser</artifactId>
+    		<version>3.0.8.RELEASE</version>
+		</dependency>
+		
+        
+		
```

Comment:
Looks good! Could you remove extra blank lines, and correct the indentation inside <dependency>? Currently, we haven't added/activated code inspection Tools to CI but once that's in place these minor things could be taken care of automatically. for now we'll abide to format manually.

Reply:
sure. Will do